Others/WiggleSort.cpp

```diff
@@ -0,0 +1,80 @@
+/*
+     Sorting an Array in a Wave Form
+    -----------
+    This is also known as Wiggle sort.It is a simplistic algorithm which which sorts
+    a given unsorted array arr[] in the following manner:
+    arr[0] < arr[1] > arr[2] < arr[3] > arr[4] <…..
```

Comment:
To also handle cases where some of the elements of the array would be duplicates, the conditions should be `<=` and `>=` instead of `<` and `>`.